polyfills/Array/from/polyfill.js

```diff
@@ -1,36 +1,104 @@
-Object.defineProperty(Array, 'from', {
-	configurable: true,
-	value: function from(source) {
-		// handle non-objects
-		if (source === undefined || source === null) {
-			throw new TypeError(source + ' is not an object');
+(function arrayFromPolyfill() {
```

Comment:
After reading http://benalman.com/news/2010/11/immediately-invoked-function-expression/ which seems to be the definitive word on iife they make it clear that an iife can have a name but made no mention of it polluting scope. So I am pretty sure it remains true.

https://kangax.github.io/nfe/#jscript-bugs -> Makes me suspect that IE8 and below could potentially have an issue.

I'll look into it. (Or will just remove the name)